charts/redpanda/values.yaml

```diff
@@ -102,25 +102,21 @@ tls:
 #
 # External access configuration
 external:
-  # Default external access value for all listeners except RPC
-  # External config doesn't apply to RPC listeners as they are never externally accessible
```

Comment:
Is this no longer true?

Reply:
This is true, but this line was moved to this location: https://github.com/redpanda-data/helm-charts/blob/d88954e09e4007346a1fbe576fa7896b16e87f4b/charts/redpanda/values.yaml#L459